roles/core/node/tasks/install_license_pkg.yml

```diff
@@ -77,4 +79,6 @@
   with_items:
     - "{{ scale_install_gpfs_adv.files.0.path | basename }}"
     - "{{ scale_install_gpfs_crypto.files.0.path | basename }}"
-  when: '"gpfs.license.std" not in scale_gpfs_license_rpm'
+  when:
```

Comment:
Above at line 75, is this variable used anywhere?

Reply:
`scale_gpfs_license_rpm`  is stored in the register variable where we will always get single license package i.e either  `adv, std, da, dm , dev . `

Reply:
We need not get crypto and gpfs.adv package if license is detected as  std and da.

Reply:
Sorry, this comment is for line 75 above, but i cannot point to it as it has not changed: 
```
    license_rpm: "{{ scale_install_gpfs_license.files.0.path | basename }}"
```